packages/design-picker/src/components/generated-design-picker.tsx

```diff
@@ -121,7 +120,12 @@ const GeneratedDesignPicker: React.FC< GeneratedDesignPickerProps > = ( {
 							<GeneratedDesignThumbnail
 								key={ design.slug }
 								slug={ design.slug }
-								thumbnailUrl={ getDesignPreviewUrl( design, { language: locale, verticalId } ) }
+								thumbnailUrl={ getDesignPreviewUrl( design, {
+									language: locale,
+									verticalId,
+									viewport_width: DEFAULT_VIEWPORT_WIDTH,
```

Comment:
We have to use the mobile viewport width on mobile. Otherwise, it would be weird

<img width="300" src="https://user-images.githubusercontent.com/13596067/174718940-35d9f40d-e01b-45e0-8ca6-738412def179.png" />

Reply:
You're right, nice catch 🙏